src/backend/overviewSite/index.html

```diff
@@ -100,15 +100,32 @@
         footer p {
             margin: 0px;
         }
+        .event-holder {
+            cursor: pointer;
 
-        img {
-            height: 160px;
```

Comment:
@naveentherocker Do not remove this, The previous dimensions for image was good.

Reply:
@geekyd The 2nd Image seems to be not perfect when it is `160px`. You face the earlier issue when it was in `160px`

Reply:
140px is not a good size. Lets not remove it.

Reply:
@geekyd Now fine. All the Images are `160px`